### DISCUSSION TITLE
Dark mode toggle button hover text

### DISCUSSION THREAD

**Discussion by normansimonr** [quarto-dev/quarto-cli#4815] [Q&A] [answered]
Hi. I am creating a website with both a dark and a light styles. When I hover over the toggle button it says "toggle dark mode" (or something similar, I'm on my phone now so I don't remember the exact language). Is there a way to customize that text?

Thanks!

**Comment by dragonstyle** [comment, marked as answer]
You can do this like:

```markdown
---
title: "My Document"
author: "Norah Jones"
date: 5/22/2022
language: 
  toggle-dark-mode: "My Custom Label"
---
```

You can also change more than one language if you'd rather (or scope it to a single language). More here:

https://quarto.org/docs/authoring/language.html#per-language-alternates

HTH!

**Reply by normansimonr** [reply]
Brilliant! Thank you so much @dragonstyle!

**Comment by normansimonr** [comment]
@dragonstyle Is `toggle-dark-mode` in the reference docs? I haven't been able to find it. If it isn't, would it make sense to add it?

**Reply by dragonstyle** [reply]
We just added this language string in Quarto 1.3, so it isn't documented yet (we don't update reference docs to use the latest version of Quarto until we are in RC or later) - we will add them (there are a few other new strings as well which we'll need to reflect as well)!

**Reply by normansimonr** [reply]
Awesome! That makes sense!
